django_file_form/models.py

```diff
@@ -94,4 +95,19 @@ def __init__(self, file_id, **kwargs):
         self.size = os.path.getsize(self.file.name)
 
     def get_values(self):
-        return dict(id=self.file_id, name=self.name)
+        return dict(id=self.file_id, name=self.name, size=self.size)
+
+
+class PlaceholderUploadedFile(object):
+
+    def __init__(self, name, size=None, placeholder=None):
```

Comment:
A `placeholder` is added to allow the specification of ID of the file because in my case `filename` is not enough to identify existing file.